File: treinos/treino016/app/src/main/java/aluno/gabriel/treino016/datamodel/UsuarioDataModel.java
package aluno.gabriel.treino016.datamodel;

public class UsuarioDataModel {
    public static final String TABELA = "USUARIOS";
    public static final String ID = "ID";
    public static final String NOME = "NOME";
    public static final String EMAIL = "EMAIL";
    public static final String SENHA = "SENHA";
    public static String query = "";

    public static String queryCreateTable(){
        query += "CREATE TABLE IF NOT EXISTS "+TABELA+" ( ";
        query += ID+" INTEGER PRIMARY KEY AUTOINCREMENT, ";
        query += NOME+" TEXT NOT NULL, ";
        query += EMAIL+" TEXT NOT NULL UNIQUE, ";
        query += SENHA+" TEXT NOT NULL ";
        query += " ) ";
        return query;
    }
}

File: treinos/treino011/app/src/main/java/aluno/gabriel/treino011/datamodel/UsuarioDataModel.kt
package aluno.gabriel.treino011.datamodel

class UsuarioDataModel {
    companion object{
        val TABELA:String = "USUARIOS"
        val ID:String = "ID"
        val NAME:String = "NOME"
        val EMAIL: String = "EMAIL"
        val PASSWORD: String = "SENHA"
        var query: String = ""

    fun queryCreateTable(): String {
        query += "CREATE TABLE IF NOT EXISTS " + TABELA + " ( "
        query += ID + " INTEGER PRIMARY KEY AUTOINCREMENT, "
        query += NAME + " TEXT NOT NULL, "
        query += EMAIL + " TEXT NOT NULL UNIQUE, "
        query += PASSWORD + " TEXT NOT NULL "
        query += " ) "
        return query
        }

    }
}